examples/NERDiffgram/step_6_whylogs.ipynb

```diff
@@ -0,0 +1,1443 @@
+    "!pip install tabulate\n",
+    "!pip install 'whylogs[viz]'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "679e660c-ade3-400a-985d-c661f1708692",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/tmp/ipykernel_75/3258265333.py:21: DeprecationWarning: Importing display from IPython.core.display is deprecated since IPython 7.14, please import from IPython.display\n",
+      "  from IPython.core.display import display, HTML\n"
+     ]
+    }
+   ],
+   "source": [
+    "import torch\n",
+    "from transformers import BertTokenizerFast, BertForTokenClassification\n",
+    "from torch.utils.data import Dataset, DataLoader\n",
+    "import pandas as pd\n",
+    "from diffgram import Project\n",
+    "from typing import List, Dict, Optional\n",
+    "import anthropic\n",
+    "import json\n",
+    "from neo4j import GraphDatabase\n",
+    "from tqdm import tqdm\n",
+    "import logging\n",
+    "import os\n",
+    "import sys\n",
+    "import boto3\n",
+    "import requests\n",
+    "import pprint\n",
+    "import json\n",
+    "from diffgram import Project\n",
+    "from llama_index.core import SimpleDirectoryReader, StorageContext\n",
+    "import pandas as pd\n",
+    "from IPython.core.display import display, HTML\n",
+    "import matplotlib as mpl\n",
+    "from tabulate import tabulate\n",
+    "import math"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "bec641c4-a6ef-4857-87ce-739bae184fc2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use os.getcwd() since __file__ is not available in interactive environments\n",
+    "current_dir = os.getcwd()\n",
+    "\n",
+    "# If your structure is such that the package is in the parent directory, compute the parent directory:\n",
+    "parent_dir = os.path.abspath(os.path.join(current_dir, '..'))\n",
+    "\n",
+    "# Add the parent directory to sys.path if it's not already there\n",
+    "if parent_dir not in sys.path:\n",
+    "    sys.path.insert(0, parent_dir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "5530d7d1-0e72-4568-a750-93bbafd823aa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import diffgram_utils as du"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a516f37e-4111-40e7-baf5-ae757d5319af",
+   "metadata": {},
+   "source": [
+    "## Connect  to Diffgram"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "77156ce4-488c-44dd-a5d9-0120c8946734",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Configuration\n",
+    "DIFFGRAM_CONFIG = {\n",
+    "    \"host\": \"http://dispatcher:8085\",\n",
+    "    \"project_string_id\": \"translucenttracker\",\n",
+    "    \"client_id\": \"LIVE__u3v8q0m7tx1p851dp0ap\",\n",
+    "    \"client_secret\": \"1qgd8as7xfcbuem6mw9j1z0xvjfmmvlagbugqr8z1g1ntypugr2ul24cce5k\"\n",
```

Comment:
Are these values coming from the local whylogs application or from Diffgram?

Reply:
local diffgram